index.html

```diff
@@ -345,18 +344,30 @@ <h2>Document Policy</h2>
     </section>
     <section id="privacy-security" class="informative">
       <h2>Privacy and Security</h2>
-      <p>
-      The primary concerns with introducing a sampling profiling for JavaScript are leakage of execution information across cross-origin execution contexts, leakage of cross-origin scripts, and potential timing attacks.
-      </p>
-      <p>
-      Implementors must take care to ensure that stack frames from cross-origin execution contexts are not leaked, even when invoked synchronously from the frame that initiates profiling. As script execution information from other contexts could be used to infer the state of these contexts (e.g. whether or not the user is logged in), the spec deems this inpermissible.
-      </p>
-      <p>
-      Including stack frames from functions defined in a cross-origin resource must be performed with caution. The contents of opaque cross-origin scripts should remain inaccessible to UAs, as the resource has not consented to inspection (even with CORP). The spec limits this by requiring all functions included in a trace to be defined in a same-origin resource, or served via CORS.
-      </p>
-      <p>
-      Lastly, timing attacks remain a concern for any API introducing a new source of high-resolution time information. The spec aims to mitigate this by requiring pages to be <a href="!HTML5#cross-origin-isolated">cross-origin isolated</a>, providing UAs with a mechanism to process-isolate pages that perform profiling. See [[?HR-Time]]'s discussion on <a href="https://www.w3.org/TR/hr-time-2/#clock-resolution">clock resolution</a>.
-      </p>
+      <p>The following sections detail some of the privacy and security choices of the API, illustrating protection strategies against various types of attacks.</p>
+      <section>
+        <h2>Cross-origin script contents</h2>
+        <p>
+        The API avoids exposing contents of cross-origin scripts by requiring all functions included via the <a>take a sample</a> algorithm to be defined in a script served with <a>CORS-same-origin</a>. Browser builtins (such as <code>performance.now()</code>) may still be included when invoked from cross-origin no-cors script (albeit with no information about the invoking script, line, or column number), although this is already made possible through manually overriding global objects.
+        </p>
+        <p>
+        As a result, the API does not expose any new insight into the contents or execution characteristics of cross-origin script, beyond what is already possible through manual instrumentation.
```

Comment:
Can we add the following sentence in the end of this line (or something similar)?
`However, User Agent is recommended to verify this holds true when they plan to expose less than 1 millisecond of sampleInterval.`

Reply:
Sounds good, done.